src/components/404/NotFoundPage.test.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { cleanup, render } from 'react-testing-library';
+import { MemoryRouter } from 'react-router-dom';
+import 'jest-dom/extend-expect';
```

Comment:
Hi Kizito, Nice implementation of the 404 page. I noticed that you're importing `jest-dom/extend-expect` and `cleanup` from react-testing-library. There is already a setup Test file that handles that. You should consider removing them. You can also remove line 8 as well.

Reply:
Fixed. Thanks for pointing this out.